tests/TestKdbx4.cpp

```diff
@@ -220,16 +322,17 @@ void TestKdbx4::testCustomData()
     publicCustomData.insert("CD1", 123);
     publicCustomData.insert("CD2", true);
     publicCustomData.insert("CD3", "abcäöü");
-    publicCustomData.insert("CD4", QByteArray::fromHex("ababa123ff"));
     db.setPublicCustomData(publicCustomData);
+    publicCustomData.insert("CD4", QByteArray::fromHex("ababa123ff"));
+    db.publicCustomData().insert("CD4", publicCustomData.value("CD4"));
     QCOMPARE(db.publicCustomData(), publicCustomData);
 
     const QString customDataKey1 = "CD1";
     const QString customDataKey2 = "CD2";
     const QString customData1 = "abcäöü";
     const QString customData2 = "Hello World";
     const int dataSize = customDataKey1.toUtf8().size() + customDataKey1.toUtf8().size() +
```

Comment:
shouldn't this be a `constexpr` ?

Reply:
QString cannot be a constexpr and dataSize depends on runtime data.

Reply:
yeah, I was talking about dataSize. Make sense

Reply:
constexpr makes most sense for global variables.